src/src/lib.rs

```diff
@@ -564,37 +556,37 @@ pub extern "C" fn roc_fx_sendRequest(roc_request: &glue::Request) -> glue::Respo
+        ErrorKind::NotFound => file_glue::WriteErr::NotFound,
+        ErrorKind::AlreadyExists => file_glue::WriteErr::AlreadyExists,
+        ErrorKind::Interrupted => file_glue::WriteErr::Interrupted,
+        ErrorKind::OutOfMemory => file_glue::WriteErr::OutOfMemory,
+        ErrorKind::PermissionDenied => file_glue::WriteErr::PermissionDenied,
+        ErrorKind::TimedOut => file_glue::WriteErr::TimedOut,
         // TODO investigate support the following IO errors may need to update API
-        std::io::ErrorKind::WriteZero => file_glue::WriteErr::WriteZero,
+        ErrorKind::WriteZero => file_glue::WriteErr::WriteZero,
         _ => file_glue::WriteErr::Unsupported,
         // TODO investigate support the following IO errors
-        // std::io::ErrorKind::FileTooLarge <- unstable language feature 
-        // std::io::ErrorKind::ExecutableFileBusy <- unstable language feature 
-        // std::io::ErrorKind::FilesystemQuotaExceeded <- unstable language feature 
-        // std::io::ErrorKind::InvalidFilename <- unstable language feature 
-        // std::io::ErrorKind::ResourceBusy <- unstable language feature 
-        // std::io::ErrorKind::ReadOnlyFilesystem <- unstable language feature 
-        // std::io::ErrorKind::TooManyLinks <- unstable language feature 
-        // std::io::ErrorKind::StaleNetworkFileHandle <- unstable language feature 
+        // std::io::ErrorKind::FileTooLarge <- unstable language feature
+        // std::io::ErrorKind::ExecutableFileBusy <- unstable language feature
+        // std::io::ErrorKind::FilesystemQuotaExceeded <- unstable language feature
+        // std::io::ErrorKind::InvalidFilename <- unstable language feature
+        // std::io::ErrorKind::ResourceBusy <- unstable language feature
+        // std::io::ErrorKind::ReadOnlyFilesystem <- unstable language feature
+        // std::io::ErrorKind::TooManyLinks <- unstable language feature
+        // std::io::ErrorKind::StaleNetworkFileHandle <- unstable language feature
         // std::io::ErrorKind::StorageFull <- unstable language feature
     }
 }
 
-fn toRocReadError(err : std::io::Error) -> file_glue::ReadErr {
-    match err.kind(){
-        std::io::ErrorKind::Interrupted => file_glue::ReadErr::Interrupted,
-        std::io::ErrorKind::NotFound => file_glue::ReadErr::NotFound,
-        std::io::ErrorKind::OutOfMemory => file_glue::ReadErr::OutOfMemory,
-        std::io::ErrorKind::PermissionDenied => file_glue::ReadErr::PermissionDenied,
-        std::io::ErrorKind::TimedOut => file_glue::ReadErr::TimedOut,
+fn toRocReadError(err: std::io::Error) -> file_glue::ReadErr {
+    match err.kind() {
+        ErrorKind::Interrupted => file_glue::ReadErr::Interrupted,
+        ErrorKind::NotFound => file_glue::ReadErr::NotFound,
+        ErrorKind::OutOfMemory => file_glue::ReadErr::OutOfMemory,
+        ErrorKind::PermissionDenied => file_glue::ReadErr::PermissionDenied,
+        ErrorKind::TimedOut => file_glue::ReadErr::TimedOut,
```

Comment:
These changes were made by `rustfmt`. I'm happy to undo them if we don't want them.